matchbox_socket/src/webrtc_socket/mod.rs

```diff
@@ -415,3 +424,18 @@ async fn wait_for_ready(channel_ready_rx: Vec<futures_channel::mpsc::Receiver<u8
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    use crate::{Error, WebRtcSocket};
+
+    #[futures_test::test]
+    async fn unreachable_server() {
+        // .invalid is a reserved tld for testing and documentation
+        let (_socket, fut) = WebRtcSocket::new("wss://invalid.xyz");
```

Comment:
Is this week another type of special domain? If so, comment should be updated

Reply:
D'oh. Didn't see the comment. That can be easily fixed.